protoc-gen-openapiv2/internal/genopenapi/helpers.go

```diff
@@ -22,6 +22,7 @@ func getUniqueFields(schemaFieldsRequired []string, fieldsRequired []string) []s
 	for j, schemaFieldRequired := range schemaFieldsRequired {
 		index = nil
 		for i, fieldRequired := range fieldsRequired {
+			i := i
```

Comment:
This wasn't technically required in this case (because of the `break` call), however in general it is good practice to avoid these sorts of issues.